books/Altele/1. Doamne atinge-ne din nou.txt

```diff
@@ -0,0 +1,23 @@
+@Title Doamne atinge-ne din nou
+@Number 1
```

Comment:
Am putea la Altele sa facem ca la Cor, fara numere, ca nu au niciun corespondent in vreo carte